src/assets/styles/variables.scss

```diff
@@ -1,3 +1,9 @@
+//media-points
+$mediaMaxWidth: 1700px;
+$mediaBp1Width: 1024px;
```

Comment:
unreadable naming (still don't understand what does it means)

propouse: 
you can use naming like this
![image](https://user-images.githubusercontent.com/98600404/212063426-61ed01c7-4d6f-44d7-a956-79abe73b1b92.png)